compat.h

```diff
@@ -0,0 +1,37 @@
+#ifndef UTCP_COMPAT_H
+#define UTCP_COMPAT_H
+
+#ifndef _MSC_VER
+  #include <unistd.h>
+#endif
+
+#ifdef _WIN32
+  #include <winsock2.h>
+#else
+  #include <sys/time.h>
+#endif
+
+#if !defined(PRINT_SIZE_T)
```

Comment:
dh besser ich entferne es und änder %zu in %llu und caste nach long long wie ihr es wie ich lese jetzt auch fürs backend macht..